src/physics/physics_model.cpp

```diff
@@ -187,7 +187,13 @@ ::convective_eigenvalues (
         }
         std::array<real,nstate_baseline_physics> baseline_eig = physics_baseline->convective_eigenvalues(baseline_conservative_soln, normal);
         for(int s=0; s<nstate_baseline_physics; ++s){
-            eig[s] += baseline_eig[s];
+            if(eig[s]!=0.0){
+                std::cout << "Error: PhysicsModel does not currently support additional convective flux terms." << std::endl; 
+                std::cout << "Aborting..." << std::endl;
```

Comment:
Would be better to use `pcout` by adding: `, mpi_communicator(MPI_COMM_WORLD)
, mpi_rank(dealii::Utilities::MPI::this_mpi_process(MPI_COMM_WORLD))
, n_mpi(dealii::Utilities::MPI::n_mpi_processes(MPI_COMM_WORLD))
, pcout(std::cout, mpi_rank==0)` to the constructor of PhysicsModel; see `src/flow_solver/flow_solver.cpp` and `src/flow_solver/flow_solver.h`

Reply:
and replace all other `std::cout` throughout this file by `pcout`

Reply:
fixed as suggested